react/components/Input/index.js

```diff
@@ -97,7 +97,7 @@ class Input extends Component {
     let prefixClasses = `${borderRadiusBase} br-0 br--left `
     let suffixClasses = `${borderRadiusBase} bl-0 br--right `
 
-    let classes = `${widthClass} ${box} ${hideDecorators} ${borderRadius} bn outline-0 `
+    let classes = `${widthClass} ${box} ${styles.hideDecorators} ${styles.noAppearance} ${borderRadius} bn outline-0 `
```

Comment:
Not sure this will scale fine if 20 or more styles are added in the css file.

Reply:
any ideas?

Reply:
Could you do a loop of some kind?

Reply:
a loop? to add all classes in `styles`? I don't think it'll be very readable

Reply:
since this is not really related with the changes I made in this PR, I think it's best to leave this issue for another PR